Exiled.API/Features/Items/Ammo.cs

```diff
@@ -16,6 +16,11 @@ namespace Exiled.API.Features.Items
     /// </summary>
     public class Ammo : Item
     {
+        /// <summary>
+        /// Specifies the maximum amount of ammo that may be held at one time.
+        /// </summary>
+        public const ushort AmmoLimit = 65535;
```

Comment:
This is a breaking change.

Reply:
How so? It's a new property that didn't exist before.

Reply:
It's not, I'm retarded